### PR TITLE
feat: support license key

### PR DESCRIPTION
**Pull request by arkbriar**
License key support.

### CODE REVIEW

File: charts/risingwave/values.yaml [hunk 1]
```diff
@@ -1389,3 +1389,21 @@ cloud:
...
+
+  ## @param secret Use existing Secret to provide license key.
+  ## If set, value of key will be ignored.
+  secret:
+    ## @param name Name of the existing Secret.
+    ##
+    name: ""
+    ## @param key Key in the Secret. Defaults to "license".
+    ##
+    key: "license"
```
Comment (fuyufjh) [inline charts/risingwave/values.yaml:1409]: Suggest to use `"licenseKey"` as Secret's key to keep aligned with https://github.com/risingwavelabs/risingwave-operator/pull/709

File: charts/risingwave/values.yaml [hunk 2]
```diff
@@ -1389,3 +1389,21 @@ cloud:
       ## This only works when endpoint is not set and region is set.
       ##
       useVPCEndpoint: false
+
+## @section license. Enterprise license key configuration for RisingWave enterprise.
+##
+
+license:
```
Comment (fuyufjh) [inline charts/risingwave/values.yaml:1396]: Ditto

Comment (arkbriar) [reply charts/risingwave/values.yaml:1396]: This is the license block, not `license key` block. I think the `key` field below stands for `licenseKey`.

**Review by fuyufjh — APPROVED** [review]
LGTM, Thanks!

**Review by wjf3121 — APPROVED** [review]

**Pull request merged**